scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/IncrementalSimulationDriver.java

```diff
@@ -133,16 +132,16 @@ public void simulateActivity(SerializedActivity activity, Duration startTime, Ac
    * Get the simulation results from the Duration.ZERO to the current simulation time point
    * @return the simulation results
    */
-  public SimulationResults getSimulationResults(){
-    return getSimulationResultsUpTo(curTime);
+  public SimulationResults getSimulationResults(Instant startTimestamp){
+    return getSimulationResultsUpTo(startTimestamp, curTime);
   }
 
   /**
    * Get the simulation results from the Duration.ZERO to a specified end time point.
```

Comment:
Does this comment still accurately describe this method's behavior? Just checking since `Duration.ZERO` doesn't appear to be used here.

Reply:
Yes. `Duration.ZERO` is the origin for simulation. The interval is `[Duration.ZERO, curTime]`.

Reply:
`startTimeStamp` is there only to set the start timestamps of activities in `SimulationResults`

Reply:
Ah gotcha! Could you please add the `@param`s for the parameters here and on the function directly above?

Reply:
Done